models/nft/ethereum/metadata/nft_ethereum_metadata.yml

```diff
@@ -69,13 +69,144 @@ models:
     columns:
       - name: contract_address
         description: "Collection contract address"
-      - name: art_blocks_project_id
+      - name: project_id
```

Comment:
Could you rename this file as nft_ethereum_metadata_schema.yml (just for consistency)

Reply:
It would also be nice to include the `models/nft/ethereum/metadata/nft_ethereum_metadata_art_platform_collections.sql` model in the schema .yml file